go.mod

```diff
@@ -10,95 +10,97 @@ require (
 	github.com/google/go-querystring v1.1.0
 	github.com/hashicorp/go-version v1.7.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/common v0.4.1
-	github.com/prometheus/prometheus v0.0.0-20190710134608-e5b22494857d
+	github.com/prometheus/common v0.55.0
+	github.com/prometheus/prometheus v0.54.1
```

Comment:
Please check whether these version in file: https://github.com/pingcap/monitoring/blob/master/dependencies.yaml should be updated.

Reply:
I tested it with Prometheus v2.27.1, it works well. So, I think we don't need to update it now.